src/components/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// Styles
+import './PageNotFound.scss';
+
+export default () => {
+  return (
+    <div className="pagenotfound-container">
```

Comment:
We already have styles for pages, why not reuse by using markdown class name ?
```suggestion
    <div className="pagenotfound-container markdown">
```